accessibility-checker-extension/src/ts/options/OptionsApp.tsx

```diff
@@ -338,12 +338,19 @@ export class OptionsApp extends React.Component<{}, OptionsAppState> {
             <aside aria-label="About Accessibility Checker Options">
             <p>
                 By default, the Accessibility Checker uses a set of rules that correspond to 
-                the most recent WCAG standards plus some additional IBM requirements. Rule sets 
-                for specific WCAG versions are also available. The rule sets are updated regularly, 
+                the WCAG standards plus some additional IBM requirements. Rule sets 
+                are available for specific WCAG versions. The rule sets are updated regularly, 
```

Comment:
```suggestion
                the WCAG standards plus some additional IBM requirements. Guidelines 
                are available for specific WCAG versions. The guidelines are updated regularly, 
```
rule sets are dated archives. IBM Accessibility vs WCAG 2.0 vs WCAG 2.2 are guidelines.